src/widgets/lv_roller.c

```diff
@@ -488,7 +488,10 @@ static void draw_main(lv_event_t * e)
 
             /*Proportional position from the middle line.
              *Will be 0 for the first option, and 1 for the last option (upscaled by << 14)*/
-            label_y_prop = (label_y_prop << 14) / (lv_obj_get_height(label) - normal_label_font->line_height);
+            lv_coord_t label_h = lv_obj_get_height(label);
+            if((label_h - normal_label_font->line_height) > 0) {
+                label_y_prop = (label_y_prop << 14) / (label_h - normal_label_font->line_height);
+            }
```

Comment:
Avoid duplication of calculations:
```c
lv_coord_t remain_h = lv_obj_get_height(label) - normal_label_font->line_height;
if(remain_h > 0) {
    label_y_prop = (label_y_prop << 14) / remain_h;
}
```

Reply:
Thank you for your advice.
I Fixed.
https://github.com/lvgl/lvgl/pull/6259/commits/fc33c1f17b8e2a5ae06524aade0b3a5209da4690